src/main/java/io/temporal/samples/polling/frequent/FrequentPollingActivityImpl.java

```diff
@@ -48,7 +49,7 @@ public String doPoll() {
 
       // heart beat and sleep for the poll duration
       try {
-        context.heartbeat("no response");
+        context.heartbeat(Optional.empty());
```

Comment:
Thanks. I imagine this could be a risk, but I hoped the serialization layer could interpreted optional correctly. 

However I am slightly confused on why the tests didn't fail,  is this related to some mocking that occurs within the test framework ? It's fixed now